webapp/app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py

```diff
@@ -299,6 +298,17 @@ def render(self, data, render_info):
             _('form.lotse.gem_haushalt-list-item-2')
         ], header_title=_('form.lotse.steuerminderungen.header-title'))
 
+    @classmethod
+    def get_redirection_info_if_skipped(cls, input_data):
+        result = super().get_redirection_info_if_skipped(input_data)
+        if result != (None, None):
+            return result
+
+        if show_person_b(input_data):
+            return StepFamilienstand.name, _l('form.lotse.skip_reason.stmind_gem_haushalt.married')
+
+        return None, None
```

Comment:
Couldn't this be `return result`? Or do you think it is better to make it explicit